infra/scripts/test-end-to-end.sh

```diff
@@ -5,11 +5,13 @@ set -o pipefail
 [[ $1 == "True" ]] && ENABLE_AUTH="true" || ENABLE_AUTH="false"
 echo "Authenication enabled : ${ENABLE_AUTH}"
 
-test -z ${GOOGLE_APPLICATION_CREDENTIALS} && GOOGLE_APPLICATION_CREDENTIALS="/etc/gcloud/service-account.json"
+test -z ${GOOGLE_APPLICATION_CREDENTIALS} && GOOGLE_APPLICATION_CREDENTIALS=""
 test -z ${SKIP_BUILD_JARS} && SKIP_BUILD_JARS="false"
 test -z ${GOOGLE_CLOUD_PROJECT} && GOOGLE_CLOUD_PROJECT="kf-feast"
 test -z ${TEMP_BUCKET} && TEMP_BUCKET="feast-templocation-kf-feast"
 test -z ${JOBS_STAGING_LOCATION} && JOBS_STAGING_LOCATION="gs://${TEMP_BUCKET}/staging-location"
+# Target SDK to run E2E tests with. Supported values: 'python', 'java', 'go'
```

Comment:
I dont think there is a plan to have full support for e2e tests or all functionality with other SDKs. Go and Java will probably only have limited functionality compared to the Python library, especially if we move towards PySpark.

Also, even if we had full compatibility between the SDKs of all functionality, we would still want to use a single e2e test framework like Pytest ideally, just to keep things consistent.

Reply:
Reverted.